lib/action_network_rest/client.rb

```diff
@@ -0,0 +1,30 @@
+module ActionNetworkRest
+  class Client < Vertebrae::API
+    attr_accessor :api_key
+
+    def initialize(options={}, &block)
+      self.api_key = options[:api_key]
+      super(options={}, &block)
+    end
+
+    def default_options
+      {
+        host: 'actionnetwork.org',
+        prefix: '/api/v2',
+        content_type: 'application/json',
+        additional_headers: {'OSDI-API-Token' => api_key},
+        user_agent: 'ruby: ActionNetworkRest'
+      }
+    end
+
+    def extract_data_from_params(params)
+      params.to_json
+    end
+
+    ## Helpers to let users do things like `an_client.person.create(params)`
+
+    def person
```

Comment:
Should this be `person` or `people` to match the API?

Reply:
:shrug: I don't have a strong opinion about it; I picked the singular since it seemed similar to what we do elsewhere, but using the same word as their API is probably the better choice.